### PR TITLE
Add `AVSampleBufferDisplayLayer` rendering mode to `VideoView`

### PR DESCRIPTION
**Pull request by hiroshihorie**
This is needed for PiP.

### CODE REVIEW

File: Sources/LiveKit/Views/InternalSampleBufferVideoRenderer.swift [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import WebRTC
+
+internal class InternalSampleBufferVideoRenderer: NativeView {
```
Comment (davidzhao) [inline Sources/LiveKit/Views/InternalSampleBufferVideoRenderer.swift:20]: What does `Internal` reference? or should we call it `AVSampleBufferDisplayLayerRenderer` ?

**Review by davidzhao — APPROVED** [review]
lg. just one question re: classname

File: Sources/LiveKit/Views/InternalSampleBufferVideoRenderer.swift (hunk 1 above)
Comment (hiroshihorie) [reply Sources/LiveKit/Views/InternalSampleBufferVideoRenderer.swift:20]: This is not exposed, the `VideoView` uses it internally as a subview.

Comment (davidzhao) [reply Sources/LiveKit/Views/InternalSampleBufferVideoRenderer.swift:20]: got it, sounds good!

**Pull request merged**